src/core/app_backend.tf

```diff
@@ -137,9 +137,10 @@ locals {
       FF_USER_AGE_LIMIT_ENABLED = 1
       FF_IO_SIGN_ENABLED        = 1
 
-      FF_ROUTING_PUSH_NOTIF                        = "BETA" # possible values are: BETA, CANARY, PROD, NONE
-      FF_ROUTING_PUSH_NOTIF_BETA_TESTER_SHA_LIST   = data.azurerm_key_vault_secret.app_backend_APP_MESSAGES_BETA_FISCAL_CODES.value
-      FF_ROUTING_PUSH_NOTIF_CANARY_SHA_USERS_REGEX = "XYZ"
+      FF_ROUTING_PUSH_NOTIF                      = "BETA" # possible values are: BETA, CANARY, PROD, NONE
```

Comment:
maybe we can set this directly to 'CANARY'

Reply:
Ok. So we need to publish io-backend first, and then apply tis change.